### ISSUE TITLE
Support GitHub Enterprise

### ISSUE THREAD

**Issue by gizumon** [marocchino/sticky-pull-request-comment#953]
Thank you for making nice feature. Please consider supporting GitHub Enterprise 🙇‍♂️ 
https://github.com/marocchino/sticky-pull-request-comment/blob/main/src/main.ts#L59

sample:
```js
    const octokit = github.getOctokit(token, {
      baseUrl: props.githubBaseUrl || 'https://api.github.com',
    })
```

**Comment by marocchino** [comment]
Released as `v2`, `v2.6.0` enjoy it. :)

**Comment by chrko** [comment]
Just as information: The action worked pretty well in GitHub enterprise instance for me. Now I have to set the `baseUrl` manually to the value of `github.api_url` instead of relying on the underlying discover the value itself.

**Comment by marocchino** [comment]
oops. sorry for that. :(
I should change default value. to `${{ env. GITHUB_API_URL }}`

**Comment by chrko** [comment]
@marocchino I'll can test the changes in the GitHub enterprise instance I have access to. :)

**Comment by marocchino** [comment]
https://github.com/marocchino/sticky-pull-request-comment/pull/955

**Comment by marocchino** [comment]
@chrko  Released as `v2`, `v2.6.1`

**Comment by voidstarstar** [comment]
It sounds like this was reverted in #957 (released as `v2.6.2`) due to #956.  Is this feature still planned to be released for a future version, such as `v3`?
